recipes/ecell4/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ecell4" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/ecell/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
Please add a note stating that the pypi version seems to be out of date.

Reply:
Thank you for your comment.
I will replace github source to pypi (instead of adding the note).
I updated the PyPI package to the latest version.